config/locales/time/es.yml

```diff
@@ -19,6 +19,7 @@ es:
     am: AM
     formats:
       event_date: '%-d de %B %Y'
+      event_timestamp_js: '%{day} de %{month} de %{year} a las %{hour}:%{minute}'
```

Comment:
there's no `dayPeriod` like there is for English, is this a 24h format?

Reply:
> there's no `dayPeriod` like there is for English, is this a 24h format?

Yep, hour is 24-hour by default in French and Spanish.

```js
new Intl.DateTimeFormat('en', { hour: 'numeric' }).formatToParts(new Date().setHours(23)).find((part) => part.type === 'hour').value
// '11'
new Intl.DateTimeFormat('es', { hour: 'numeric' }).formatToParts(new Date().setHours(23)).find((part) => part.type === 'hour').value
// '23'
new Intl.DateTimeFormat('fr', { hour: 'numeric' }).formatToParts(new Date().setHours(23)).find((part) => part.type === 'hour').value
// '23'
```

Reply:
I might want to double-check how this is interpreted, since while `en` returns 12-hour for me, I'd wonder if someone in an international English locale could see something different. Or maybe "en" is globally understand as "en-US".

```js
new Intl.DateTimeFormat('en', { hour: 'numeric' }).formatToParts(new Date().setHours(23)).find((part) => part.type === 'hour').value
// '11'
new Intl.DateTimeFormat('en-GB', { hour: 'numeric' }).formatToParts(new Date().setHours(23)).find((part) => part.type === 'hour').value
// '23'
```

Reply:
Is there a way to read the `hour12` default value and use that? I feel like that's how we could best respect the user's preferences?

Otherwise I think `DateTimeFormat#format` is probably acceptable

Reply:
> Is there a way to read the `hour12` default value and use that? I feel like that's how we could best respect the user's preferences?

I may be misunderstanding, but it should be respecting the user's preference already as long as we don't set the setting explicitly. While we expect 12-hour `en-US` and 24-hour `es-MX` and `fr-CA`, the implementation should tolerate any variation (e.g. 24-hour `en-GB`), albeit a bit hackishly.

> Otherwise I think `DateTimeFormat#format` is probably acceptable

I'll try to get some UX feedback on this, since it seems to (a) be much simpler to implement and (b) fully leverages the browser native behavior to produce a sensible output.

Reply:
I went a different direction in [bba1c1d](https://github.com/18F/identity-idp/pull/5967/commits/bba1c1dea3067f5a7ffff32e3b2583a2aea9a0a7), to more closely follow the existing precedent of Ruby formatted time strings, where [English includes the period flag and a 12h-formatted hour](https://github.com/18F/identity-idp/blob/24f5694fb52d4db48e7c81e97598133b1e7ee69d/config/locales/time/en.yml#L22), while [French](https://github.com/18F/identity-idp/blob/24f5694fb52d4db48e7c81e97598133b1e7ee69d/config/locales/time/fr.yml#L22) and [Spanish](https://github.com/18F/identity-idp/blob/24f5694fb52d4db48e7c81e97598133b1e7ee69d/config/locales/time/es.yml) do not. I used the presence of a `%{day_period}` tag to explicitly control whether the time should be formatted as 12h or 24h.